Example/Messaging/Sample/Sample/Identity.swift

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 Google
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+
+public final class Identity: ObservableObject {
+  @Published public var instanceID = ""
+  @Published public var installationID = ""
+  @Published public var token = ""
```

Comment:
Nit: can you please add some documentation here as to what each one is? Mostly targeting a user who is new to Messaging and has never heard of FIS/IID.

Reply:
done. also realize only one ID is used, so remove installationID parameter.

Reply:
done.